include/wlr/types/wlr_wl_shell.h

```diff
@@ -26,13 +33,58 @@ struct wlr_wl_shell_surface {
 	uint32_t ping_serial;
 	struct wl_event_source *ping_timer;
 
+	bool toplevel;
+	struct wlr_wl_shell_surface_transient_state *transient_state;
+	char *title;
+	char *class_;
```

Comment:
Why the underscore?

Reply:
It was there in the argument list. I guess it breaks the build if using wlroots from C++.

Reply:
I don't care about C++, change it.